<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="date" content="2023-02-15" />
  <title>BatchQC Report</title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="/home/pramod/R/x86_64-pc-linux-gnu-library/4.2/rmarkdown/rmarkdown/templates/html_vignette/resources/vignette.css" type="text/css" />
  <style type="text/css">body{max-width: 100%;}</style>
  <script src="libs/header-attrs-2.18.1/header-attrs.js"></script>

<link href="libs/bootstrap/css/bootstrap.min.css" rel="stylesheet" media="screen">
<link href="libs/bootstrap/css/bootstrap-responsive.min.css" rel="stylesheet" media="screen">
<link href="libs/bootstrap/css/user.css" rel="stylesheet" media="screen">
</head>
<body data-spy="scroll" data-target=".sidebar" data-offset="50">
	<div id="wrap">
		<header class="subhead" id="topheader">
			<div class="container">
				<h1>BatchQC Report</h1>
				<p class="lead">
					Tests for checking Batch Effects
				</p>
			</div>
		</header>

		<div class="container">
			<div class="row">

				<div class="span3 sidebar">

					<div class="affixdiv" data-spy="affix" data-offset-top="400" data-clampedwidth=".sidebar">
						<div class="well" style="padding: 8px 0;">
							<ul class="nav nav-list usermenu">
								<li class="nav-header">Contents</li>
								<li class="active"><a href="#topheader"><i class="icon-bullhorn  icon-white"></i> BatchQC Report </a></li>
<ul>
<li><a href="#summary" id="toc-summary">Summary</a>
<ul>
<li><a href="#confounding" id="toc-confounding">Confounding</a></li>
<li><a href="#variation-analysis" id="toc-variation-analysis">Variation
Analysis</a></li>
<li><a href="#p-value-analysis" id="toc-p-value-analysis">P-value
Analysis</a></li>
</ul></li>
<li><a href="#differential-expression"
id="toc-differential-expression">Differential Expression</a>
<ul>
<li><a href="#expression-plot" id="toc-expression-plot">Expression
Plot</a></li>
<li><a href="#limma" id="toc-limma">LIMMA</a></li>
</ul></li>
<li><a href="#median-correlations" id="toc-median-correlations">Median
Correlations</a></li>
<li><a href="#heatmaps" id="toc-heatmaps">Heatmaps</a>
<ul>
<li><a href="#heatmap" id="toc-heatmap">Heatmap</a></li>
<li><a href="#sample-correlations" id="toc-sample-correlations">Sample
Correlations</a></li>
</ul></li>
<li><a href="#circular-dendrogram" id="toc-circular-dendrogram">Circular
Dendrogram</a></li>
<li><a href="#pca-principal-component-analysis"
id="toc-pca-principal-component-analysis">PCA: Principal Component
Analysis</a>
<ul>
<li><a href="#pca" id="toc-pca">PCA</a></li>
<li><a href="#explained-variation"
id="toc-explained-variation">Explained Variation</a></li>
</ul></li>
<li><a href="#shape" id="toc-shape">Shape</a></li>
<li><a href="#combat-plots" id="toc-combat-plots">Combat Plots</a></li>
<li><a href="#sva" id="toc-sva">SVA</a>
<ul>
<li><a href="#summary-1" id="toc-summary-1">Summary</a></li>
</ul></li>
</ul>
							</ul>
						</div>
					</div>

				</div>
<div id="header">
<h1 class="title">BatchQC Report</h1>
<h3 class="date">2023-02-15</h3>
</div>
<!--
<div id="TOC">
<ul>
<li><a href="#summary" id="toc-summary">Summary</a>
<ul>
<li><a href="#confounding" id="toc-confounding">Confounding</a></li>
<li><a href="#variation-analysis" id="toc-variation-analysis">Variation
Analysis</a></li>
<li><a href="#p-value-analysis" id="toc-p-value-analysis">P-value
Analysis</a></li>
</ul></li>
<li><a href="#differential-expression"
id="toc-differential-expression">Differential Expression</a>
<ul>
<li><a href="#expression-plot" id="toc-expression-plot">Expression
Plot</a></li>
<li><a href="#limma" id="toc-limma">LIMMA</a></li>
</ul></li>
<li><a href="#median-correlations" id="toc-median-correlations">Median
Correlations</a></li>
<li><a href="#heatmaps" id="toc-heatmaps">Heatmaps</a>
<ul>
<li><a href="#heatmap" id="toc-heatmap">Heatmap</a></li>
<li><a href="#sample-correlations" id="toc-sample-correlations">Sample
Correlations</a></li>
</ul></li>
<li><a href="#circular-dendrogram" id="toc-circular-dendrogram">Circular
Dendrogram</a></li>
<li><a href="#pca-principal-component-analysis"
id="toc-pca-principal-component-analysis">PCA: Principal Component
Analysis</a>
<ul>
<li><a href="#pca" id="toc-pca">PCA</a></li>
<li><a href="#explained-variation"
id="toc-explained-variation">Explained Variation</a></li>
</ul></li>
<li><a href="#shape" id="toc-shape">Shape</a></li>
<li><a href="#combat-plots" id="toc-combat-plots">Combat Plots</a></li>
<li><a href="#sva" id="toc-sva">SVA</a>
<ul>
<li><a href="#summary-1" id="toc-summary-1">Summary</a></li>
</ul></li>
</ul>
</div>
-->
<div class="span9 content">
<div id="summary" class="section level1">
<h1>Summary</h1>
<div id="confounding" class="section level2">
<h2>Confounding</h2>
<div id="number-of-samples-in-each-batch-and-condition"
class="section level3">
<h3>Number of samples in each Batch and Condition</h3>
<table style="width:83%;">
<colgroup>
<col width="25%" />
<col width="29%" />
<col width="29%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Batch 2020_dataset</th>
<th align="center">Batch 2021_dataset</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>Condition 1</strong></td>
<td align="center">90</td>
<td align="center">180</td>
</tr>
</tbody>
</table>
</div>
<div id="measures-of-confounding-between-batch-and-condition"
class="section level3">
<h3>Measures of confounding between Batch and Condition</h3>
<table style="width:99%;">
<colgroup>
<col width="44%" />
<col width="36%" />
<col width="18%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Standardized Pearson Correlation Coefficient</th>
<th align="center">Cramer’s V</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>Confounding Coefficients (0=no confounding,
1=complete confounding)</strong></td>
<td align="center">NA</td>
<td align="center">NA</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="variation-analysis" class="section level2">
<h2>Variation Analysis</h2>
<div id="variation-explained-by-batch-and-condition"
class="section level3">
<h3>Variation explained by Batch and Condition</h3>
<p><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-10-1.png" /><!-- --></p>
<table style="width:82%;">
<colgroup>
<col width="19%" />
<col width="34%" />
<col width="16%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Full (Condition+Batch)</th>
<th align="center">Condition</th>
<th align="center">Batch</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>Min.</strong></td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">0</td>
</tr>
<tr class="even">
<td align="center"><strong>1st Qu.</strong></td>
<td align="center">12.66</td>
<td align="center">0</td>
<td align="center">12.66</td>
</tr>
<tr class="odd">
<td align="center"><strong>Median</strong></td>
<td align="center">44.53</td>
<td align="center">0</td>
<td align="center">44.53</td>
</tr>
<tr class="even">
<td align="center"><strong>Mean</strong></td>
<td align="center">36.61</td>
<td align="center">0</td>
<td align="center">36.61</td>
</tr>
<tr class="odd">
<td align="center"><strong>3rd Qu.</strong></td>
<td align="center">54.48</td>
<td align="center">0</td>
<td align="center">54.48</td>
</tr>
<tr class="even">
<td align="center"><strong>Max.</strong></td>
<td align="center">82.09</td>
<td align="center">0</td>
<td align="center">82.09</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="p-value-analysis" class="section level2">
<h2>P-value Analysis</h2>
<div
id="distribution-of-batch-and-condition-effect-p-values-across-genes"
class="section level3">
<h3>Distribution of Batch and Condition Effect p-values Across
Genes</h3>
<table>
<colgroup>
<col width="27%" />
<col width="7%" />
<col width="10%" />
<col width="9%" />
<col width="10%" />
<col width="13%" />
<col width="9%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Min.</th>
<th align="center">1st Qu.</th>
<th align="center">Median</th>
<th align="center">Mean</th>
<th align="center">3rd Qu.</th>
<th align="center">Max.</th>
<th align="center">Ps&lt;0.05</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>Batch P-values</strong></td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">0.07678</td>
<td align="center">2.673e-09</td>
<td align="center">0.9726</td>
<td align="center">0.8421</td>
</tr>
<tr class="even">
<td align="center"><strong>Condition P-values</strong></td>
<td align="center">1</td>
<td align="center">1</td>
<td align="center">1</td>
<td align="center">1</td>
<td align="center">1</td>
<td align="center">1</td>
<td align="center">0</td>
</tr>
</tbody>
</table>
<p><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-13-1.png" /><!-- --></p>
<p><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-14-1.png" /><!-- --></p>
</div>
</div>
</div>
<div id="differential-expression" class="section level1">
<h1>Differential Expression</h1>
<div id="expression-plot" class="section level2">
<h2>Expression Plot</h2>
<p>Boxplots for all values for each of the samples and are colored by
batch membership.</p>
<p><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-16-1.png" /><!-- --></p>
</div>
<div id="limma" class="section level2">
<h2>LIMMA</h2>
</div>
</div>
<div id="median-correlations" class="section level1">
<h1>Median Correlations</h1>
<p>This plot helps identify outlying samples. <img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-19-1.png" /><!-- --></p>
</div>
<div id="heatmaps" class="section level1">
<h1>Heatmaps</h1>
<div id="heatmap" class="section level2">
<h2>Heatmap</h2>
<p>This is a heatmap of the given data matrix showing the batch effects
and variations with different conditions. <img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-21-1.png" /><!-- --></p>
</div>
<div id="sample-correlations" class="section level2">
<h2>Sample Correlations</h2>
<p>This is a heatmap of the correlation between samples. <img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-22-1.png" /><!-- --></p>
</div>
</div>
<div id="circular-dendrogram" class="section level1">
<h1>Circular Dendrogram</h1>
<p>This is a Circular Dendrogram of the given data matrix colored by
batch to show the batch effects. <img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-24-1.png" /><!-- --></p>
</div>
<div id="pca-principal-component-analysis" class="section level1">
<h1>PCA: Principal Component Analysis</h1>
<div id="pca" class="section level2">
<h2>PCA</h2>
<p>This is a plot of the top two principal components colored by batch
to show the batch effects. <img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-26-1.png" /><!-- --></p>
</div>
<div id="explained-variation" class="section level2">
<h2>Explained Variation</h2>
<table>
<colgroup>
<col width="4%" />
<col width="12%" />
<col width="12%" />
<col width="14%" />
<col width="14%" />
<col width="11%" />
<col width="14%" />
<col width="14%" />
</colgroup>
<thead>
<tr class="header">
<th align="center"> </th>
<th align="center">Proportion of Variance (%)</th>
<th align="center">Cumulative Proportion of Variance (%)</th>
<th align="center">Percent Variation Explained by Either Condition or
Batch</th>
<th align="center">Percent Variation Explained by Condition</th>
<th align="center">Condition Significance (p-value)</th>
<th align="center">Percent Variation Explained by Batch</th>
<th align="center">Batch Significance (p-value)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><strong>PC1</strong></td>
<td align="center">40.97</td>
<td align="center">40.97</td>
<td align="center">88.4</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">88.4</td>
<td align="center">0</td>
</tr>
<tr class="even">
<td align="center"><strong>PC2</strong></td>
<td align="center">11.04</td>
<td align="center">52.01</td>
<td align="center">2.5</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">2.5</td>
<td align="center">0.00988</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC3</strong></td>
<td align="center">8.503</td>
<td align="center">60.51</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.9342</td>
</tr>
<tr class="even">
<td align="center"><strong>PC4</strong></td>
<td align="center">6.51</td>
<td align="center">67.02</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.9338</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC5</strong></td>
<td align="center">5.964</td>
<td align="center">72.99</td>
<td align="center">0.4</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0.4</td>
<td align="center">0.3041</td>
</tr>
<tr class="even">
<td align="center"><strong>PC6</strong></td>
<td align="center">4.34</td>
<td align="center">77.33</td>
<td align="center">0.7</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0.7</td>
<td align="center">0.1861</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC7</strong></td>
<td align="center">3.88</td>
<td align="center">81.21</td>
<td align="center">0.5</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0.5</td>
<td align="center">0.2601</td>
</tr>
<tr class="even">
<td align="center"><strong>PC8</strong></td>
<td align="center">3.319</td>
<td align="center">84.53</td>
<td align="center">0.2</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0.2</td>
<td align="center">0.5098</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC9</strong></td>
<td align="center">3.22</td>
<td align="center">87.75</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.8036</td>
</tr>
<tr class="even">
<td align="center"><strong>PC10</strong></td>
<td align="center">2.75</td>
<td align="center">90.49</td>
<td align="center">0.1</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0.1</td>
<td align="center">0.6969</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC11</strong></td>
<td align="center">2.191</td>
<td align="center">92.69</td>
<td align="center">2.3</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">2.3</td>
<td align="center">0.01178</td>
</tr>
<tr class="even">
<td align="center"><strong>PC12</strong></td>
<td align="center">1.735</td>
<td align="center">94.42</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.9137</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC13</strong></td>
<td align="center">1.321</td>
<td align="center">95.74</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.7177</td>
</tr>
<tr class="even">
<td align="center"><strong>PC14</strong></td>
<td align="center">1.089</td>
<td align="center">96.83</td>
<td align="center">0.3</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0.3</td>
<td align="center">0.4101</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC15</strong></td>
<td align="center">0.9324</td>
<td align="center">97.76</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.9878</td>
</tr>
<tr class="even">
<td align="center"><strong>PC16</strong></td>
<td align="center">0.8094</td>
<td align="center">98.57</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.7266</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC17</strong></td>
<td align="center">0.6174</td>
<td align="center">99.19</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.7761</td>
</tr>
<tr class="even">
<td align="center"><strong>PC18</strong></td>
<td align="center">0.4565</td>
<td align="center">99.65</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.9957</td>
</tr>
<tr class="odd">
<td align="center"><strong>PC19</strong></td>
<td align="center">0.3541</td>
<td align="center">100</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">1</td>
<td align="center">0</td>
<td align="center">0.7514</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="shape" class="section level1">
<h1>Shape</h1>
<p>This is a heatmap plot showing the variation of gene expression mean,
variance, skewness and kurtosis between samples grouped by batch to see
the batch effects variation <img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-29-1.png" /><!-- --></p>
<pre><code>## Note: Sample-wise p-value is calculated for the variation across samples on the measure across genes. Gene-wise p-value is calculated for the variation of each gene between batches on the measure across each batch. If the data is quantum normalized, then the Sample-wise measure across genes is same for all samples and Gene-wise p-value is a good measure.</code></pre>
</div>
<div id="combat-plots" class="section level1">
<h1>Combat Plots</h1>
<p>This is a plot showing whether parametric or non-parameteric prior is
appropriate for this data. It also shows the Kolmogorov-Smirnov test
comparing the parametric and non-parameteric prior distribution.</p>
<pre><code>## Found 2 batches
## Adjusting for 0 covariate(s) or covariate level(s)
## Standardizing Data across genes
## Fitting L/S model and finding priors</code></pre>
<p><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-31-1.png" /><!-- --><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-31-2.png" /><!-- --><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-31-3.png" /><!-- --><img
src="batchqc_report_olink_2020_2021_files/figure-html/unnamed-chunk-31-4.png" /><!-- --></p>
<pre><code>## Batch mean distribution across genes: Normal vs Empirical distribution
## Two-sided Kolmogorov-Smirnov test
## Selected Batch: 1
## Statistic D = 0.1609
## p-value = 0.6517
## 
## 
## Batch Variance distribution across genes: Inverse Gamma vs Empirical distribution
## Two-sided Kolmogorov-Smirnov test
## Selected Batch: 1
## Statistic D = 0.6842
## p-value = 0.0001562Note: The non-parametric version of ComBat takes much longer time to run and we recommend it only when the shape of the non-parametric curve widely differs such as a bimodal or highly skewed distribution. Otherwise, the difference in batch adjustment is very negligible and parametric version is recommended even if p-value of KS test above is significant.</code></pre>
</div>
<div id="sva" class="section level1">
<h1>SVA</h1>
<div id="summary-1" class="section level2">
<h2>Summary</h2>
<pre><code>## Number of Surrogate Variables found in the given data: 1</code></pre>
</div>
</div>
</div>
     </div>
   </div>
  </div>
   <!-- FOOTER  -->
   <div id="footer">
		<div class="container">
			<p class="muted credit">Questions? Kindly contact <code>Solaiappan Manimaran</code> <a href="mailto:mani2012@bu.edu?subject=BatchQC question"><code>mani2012 [at] bu.edu</code></a> using the subject heading <code>BatchQC question</code>. </p>
		</div>
	</div>

	<script src="libs/bootstrap/javascript/jquery-1.10.1.min.js"></script>
	<script src="libs/bootstrap/javascript/bootstrap.min.js"></script>
	<script src="libs/bootstrap/javascript/user.js"></script>
  <script>
  $( "ul" ).addClass( "nav nav-list usermenu" );
  </script>

</body>
</html>
