---
title: "R Notebook"
output: html_notebook
---


## Asemble clinical data
```{r}
## Read Gene expression datasets

rnaseq_2020 <- readr::read_tsv(paste0(base_path_raw, "2020LD_rnaseq.tsv")) %>%
  dplyr::left_join(., clinical_data) %>%
  dplyr::filter(Timepoint <= 120)

rnaseq_2021 <- readr::read_tsv(paste0(base_path_raw, "2021LD_rnaseq.tsv")) %>%
  dplyr::left_join(., clinical_data) %>%
  dplyr::filter(Timepoint <= 120)


```
## Identify Features with coefficient of variation> 30%
```{r}
rnaseq_2020_CV <- rnaseq_2020 %>%
  group_by(versioned_ensembl_gene_id) %>%
     summarise(mean = mean(tpm), 
               median = median(tpm), 
               cv= cmstatr::cv(tpm, na.rm = T)) %>%
  filter(cv >= 0.3, median >= 5)

## Result:: Gene list filtered to 12,548 genes for 2020 dataset

rnaseq_2021_CV <- rnaseq_2021 %>%
  group_by(versioned_ensembl_gene_id) %>%
     summarise(mean = mean(tpm), 
               median = median(tpm), 
               cv= cmstatr::cv(tpm, na.rm = T)) %>%
  filter(cv >= 0.3, median >= 5)

## Result:: Gene list filtered to 16,830 genes for 2021 dataset

```


## Asemble clinical data
```{r}
rnaseq_2020_selected = rnaseq_2020 

rnaseq_2021_selected = rnaseq_2021 %>% 
  filter(versioned_ensembl_gene_id %in% rnaseq_2021_CV$versioned_ensembl_gene_id) %>%
  filter(versioned_ensembl_gene_id %in%  unique(rnaseq_2020_selected$versioned_ensembl_gene_id))

rnaseq_2020_selected <- rnaseq_2020_selected %>%
  filter(versioned_ensembl_gene_id %in%  unique(rnaseq_2021_selected$versioned_ensembl_gene_id))

rnaseq_selected <- rbind(rnaseq_2020_selected, rnaseq_2021_selected)



```

## Identify batch effects
```{r}

rnaseq_selected.df <- rnaseq_selected %>%
  dplyr::select(specimen_id, versioned_ensembl_gene_id, tpm) %>%
  pivot_wider(names_from = specimen_id, values_from = tpm) 

rnaseq_selected.matrix <-  rnaseq_selected.df %>%
  column_to_rownames("versioned_ensembl_gene_id") %>%
  as.matrix()

clinical_data_selected = clinical_data %>%
  filter(specimen_id %in% colnames(rnaseq_selected.matrix))
  
batch = clinical_data_selected$dataset

setwd("/home/pramod/Documents/GitHub/gitlab/immunoprofiling-reproducibility/scripts/batch_correction/batch_qc/rnaseq/unprocessed")
## Run BatcQC
batchQC(rnaseq_selected.matrix, 
        batch= batch,
        report_file="batchqc_report_rnaseq_2020_2021.html", report_dir=".", 
        report_option_binary="111111111",
        view_report=FALSE, 
        interactive=FALSE, 
        batchqc_output=TRUE
        )
```
### Run ComBat on rnaseq data
RNASeq data has been log-transformed, so can run ComBat (assuming near-normality of distribution)
```{r}
rnaseq_adjust<-ComBat(dat=rnaseq_selected.matrix, 
                     batch=batch)

rnaseq_adjust_df <- rnaseq_adjust %>%
  as.data.frame() %>%
  rownames_to_column("versioned_ensembl_gene_id")
```

```{r}
setwd("/home/pramod/Documents/GitHub/gitlab/immunoprofiling-reproducibility/scripts/batch_correction/batch_qc/rnaseq/batch_corrected")

## Run BatcQC
batchQC(rnaseq_adjust, 
        batch= batch,
        report_file="batchqc_report_rnaseq_2020_2021_batch_corrected.html", report_dir=".", 
        report_option_binary="111111111",
        view_report=FALSE, 
        interactive=FALSE, 
        batchqc_output=TRUE
        )
```


## Save dataframe 
```{r}
write_tsv(rnaseq_adjust_df, 
          file = paste0(base_path_processed, "rnaseq_batch_corrected.tsv"))

write_tsv(rnaseq_selected.df, 
          file = paste0(base_path_pre_processed, "rnaseq_batch_not_corrected.tsv"))
```
